### PR TITLE
Developed ERD

### PR DESCRIPTION
**Pull request by SajjadAhmad14**
Add a diagram for User
Add a diagram for Friendship
Add a diagram for Post
Add a diagram for Comment
Add a diagram for Like

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made excellent progress in this milestone. I appreciate the proper labeling of primary and foreign keys in your diagram.
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] User table should have the `image_link` column ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/imglink.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))
- Replace the `Field` column with `image_link` or `avatar`
- [x] Friendship table should have `status` or `confirmed` column ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/status.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))

- [x] The `Comment` table should have a `content` or `body` column where the main comment will be stored.

- [x] The `Post` table should have a `content` or `body` column where the main post will be stored.

- [x] GitHub Actions is set up, but it's not checking/running on your PR page. Kindly make your linter checks run and ensure that all checks are passed on your PR page.


**[ OPTIONAL CHANGES ]**

- [x] Kindly separate your relationships (connections) so that each can be distinctively represented and interpreted. Do not join their lines together.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Congratulations team! Your ERD looks good. You may proceed to the next milestone.

_Happy coding!_ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**
